intl/msg_hash_us.h

```diff
@@ -1542,7 +1542,7 @@ MSG_HASH(
    )
 MSG_HASH(
    MENU_ENUM_SUBLABEL_JOYPAD_DRIVER,
-   "Controller driver to use."
+   "Controller driver to use. (Restart required)"
```

Comment:
s/Controller driver to use. (Restart required)/Controller driver to use (Restart required)./